src/components/mapPageComponents/logic/init.ts

```diff
@@ -0,0 +1,145 @@
+import {
+  createNeighborhoodMarker,
+  setBlocksStyle,
+  setNeighborhoodsStyle,
+  setPrivateStreetsStyle,
+  setSitesStyle,
+} from './style';
+import {
+  BlockGeoData,
+  MapViews,
+  NeighborhoodGeoData,
+  SiteGeoData,
+} from '../ducks/types';
+import { addTreePopupOnClick, addZoomToClickedNeighborhood } from './event';
+import { BasicTreeInfo } from '../../treePopup';
+import { message } from 'antd';
+import { CheckboxValueType } from 'antd/es/checkbox/Group';
+
+// Logic for creating and setting up data layers/markers
+
+/**
+ * Creates the private streets layer and sets its initial style, returns the layer.
+ * @param map the map to add the layer to
+ */
+export function initPrivateStreets(map: google.maps.Map): google.maps.Data {
+  const privateStreetsLayer = new google.maps.Data({ map });
+  // Loads the objects into the layer
+  privateStreetsLayer.loadGeoJson(
+    'https://raw.githubusercontent.com/florisdobber/SFTT-map-test/master/private_streets.json',
```

Comment:
Can we move this into the bundle? Or host this on S3 instead?

Reply:
#101